### PR TITLE
PBI-10-View API Monitor Detail-Frontend

### LINKED ISSUE #33: API Monitor Detail

### PR DESCRIPTION
**Pull request by luthfifahlevi**
### Describe your changes
- Create tests for the chart components and API Monitor Detail page
- Create the chart components: success rate chart and response time chart
- Create the API Monitor Detail page
- Integrate with the backend

### Backlog name
[MON-54 View API monitor detail frontend](https://monapi.atlassian.net/browse/MON-54)

### Checklist
- [x] I have performed a self-review of my code
- [x] Code successfully run on local
- [x] Feature / changes tested on local
- [x] No failing unit test
- [ ] Passed UAT Test
- [x] Sonarqube tested 
- [ ] Required any changes to environment variable

### Example:
image

### CODE REVIEW

File: src/routes/view_api_monitor_detail/index.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  useEffect(() => {
+    axios.get(`${BASE_URL}/monitor/${id}/detail`, {
+      params:{
+        range: `${selectValue}`
+      },
+      headers: {
+        Authorization: `Token ${getUserToken()}`
+      } 
+    }).then((response) => {
+      console.log('response select time:', response.data)
```
Comment (andrew-susanto) [inline src/routes/view_api_monitor_detail/index.js]:
```suggestion
```

File: src/routes/view_api_monitor_detail/index.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        Authorization: `Token ${getUserToken()}`
+      } 
+    }).then(() => {
+      route(ROUTE.DASHBOARD)
+      setIsLoadingDelete(false)
+    })
+  }
+
+  useEffect(() => {
+    axios.get(`${BASE_URL}/monitor/${id}/detail`, {
```
Comment (andrew-susanto) [inline src/routes/view_api_monitor_detail/index.js]:
```suggestion
    axios.get(`${BASE_URL}/monitor/${id}/`, {
```

ini kaykanya url nya bakal kayak gini aja fi

**Comment by andrew-susanto** [comment]
Buat test yang chart != null mungkin bisa panggil render 2x fii, pake data yang berbeda

**Comment by luthfifahlevi** [comment]
> Buat test yang chart != null mungkin bisa panggil render 2x fii, pake data yang berbeda

kucoba di lokal tetap gk ke cover hmm

**Review by luckysusanto — APPROVED** [review]

**Review by HugoIr — APPROVED** [review]
Pengerjaan sudah menerapkan TDD dan test case yang dibuat juga hampir mengcover fitur yang dibuat.

**Review by andrew-susanto — APPROVED** [review]

**Pull request merged**